views/address.tmpl

```diff
@@ -115,8 +116,10 @@
                                 <span data-age="{{.Time}}"></span>
                             {{end}}
                             </td>
-                            <td>{{.Confirmations}}</td>
+                            <span data-confirmation-block-height="{{index $heights $i}}"></span>
```

Comment:
Instead of using a separate span element `data-confirmation-block-height="{{index $heights $i}}` should be set _directly_ on the element that wraps the value to be updated. This saves a bit of page weight and eliminates bug surface area.